lib/default.nix

```diff
@@ -0,0 +1,18 @@
+let
+  inherit (import <microgram/sdk.nix>) pkgs lib;
+  inherit (lib) mapAttrsToList concatStringsSep;
+in rec {
+
+  exportSessionVariables = mapcats (k: v: "export ${k}=${v}") {
+
+    TZDIR = "${pkgs.tzdata}/share/zoneinfo";
```

Comment:
`pkgs` here is a direct reference to nixpkgs, that's mostly sticking out for debugging purposes